### PR TITLE
Remove old Dev UI: Extension and Core

### LINKED ISSUE #34206: Removal of the old Dev UI
### Description

Migrate the missing extensions and remove the old dev-ui

### Analysis
(links to analysis docs containing architecture design work, requirements gathering, etc)

**Core Dev UI to be cleaned:**
- [x] Extensions #35317
- [x] Configuration #35163
- [x] Dev Services #35155
- [x] Continuous Testing #35154
- [x] Log streaming  #35164
- [x] Build Info #35153

**Core Extensions to be cleaned:**

- [x] arc #34971
- [x] cache #34972
- [x] container-image #34973
- [x] datasource #34974 
- [x] flyway #35073
- [x] hibernate-orm #35001
- [x] hibernate-orm-elastic #35002
- [x] kubernetes/vanilla #34983
- [x] liquibase #34984
- [x] micrometer #34982
- [x] qute #34981
- [x] resteasy-reactive/quarkus-resteasy-reactive #35046
- [x] scheduler #34980
- [x] smallrye-fault-tolerance #34979
- [x] smallrye-graphql #34976
- [x] smallrye-health #34977
- [x] smallrye-metrics #34978
- [x] smallrye-openapi #34975
- [x] smallrye-reactive-messaging-rabbitmq #35016
- [x] smallrye-reactive-messaging #35021
- [x] kubernetes/openshift #35000
- [x] resteasy-reactive/rest-client-reactive #35047
- [x] infinispan-client #35024
- [x] oidc #35048
- [x] grpc #35049
- [x] kafka-client #35050
- [ ] kafka-streams

**Quarkiverse extensions to be cleaned:**

- [x] quarkus-neo4j https://github.com/quarkiverse/quarkus-neo4j/pull/189
- [x] quarkus-operator-sdk https://github.com/quarkiverse/quarkus-operator-sdk/pull/559
- [x] quarkus-dashbuilder https://github.com/quarkiverse/quarkus-dashbuilder/pull/58
- [x] quarkus-jpastreamer https://github.com/quarkiverse/quarkus-jpastreamer/pull/81
- [x] quarkus-minio https://github.com/quarkiverse/quarkus-minio/pull/252
- [x] quarkus-ngrok https://github.com/quarkiverse/quarkus-ngrok/pull/74
- [ ] quarkus-jef 
- [ ] quarkus-cxf
- [ ] quarkus-logging-manager

### PR DESCRIPTION
**Pull request by phillip-kruger**
This removes the final bit of the Old Dev UI

fix #34206

### CODE REVIEW

**Comment by phillip-kruger** [comment]
Once this rebase on the arc PR we should be ok

**Comment by gsmet** [comment]
Awesome \o/

In the end, did we migrate the Kafka Streams Dev UI? I don't remember seeing it but I might have missed it.

File: extensions/vertx-http/deployment/pom.xml [hunk 1]
```diff
@@ -65,59 +64,7 @@
...
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.webjars</groupId>
-            <artifactId>font-awesome</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.webjars</groupId>
-            <artifactId>jquery</artifactId>
```
[the remark this answers is not in the document]
Comment (phillip-kruger) [reply extensions/vertx-http/deployment/pom.xml:91]: yes. the idea is to remove the ui from the logging manager extension, and work on enable the logging in prod mode as part of an effort to allow certain things in prod mode

**Comment by phillip-kruger** [comment]
re: kafka streams, no not migrated. i'll look at it at some point

**Comment by gsmet** [comment]
I would recommend to at least run `mvn -Dquickly` locally before pushing.

**Comment by phillip-kruger** [comment]
I did ?!? Let me check again

**Comment by phillip-kruger** [comment]
I might have to clear my m2 cache

**Comment by phillip-kruger** [comment]
@gsmet - Yea so removing m2 cache recreated this on local. Building fine on local now, lets see what CI says.

**Review by cescoffier — APPROVED** [review]
LGTM

Once this is merged, we need to check out the virtual server (and maybe it will fix funqy in dev mode with the dev UI)

**Comment by phillip-kruger** [comment]
Thanks @cescoffier ! Seems I need to fix a quickstart first. I'll look at that in the morning

**Comment by gsmet** [comment]
@phillip-kruger let's do that in another PR but given OptaPlanner is not part of the Platform anymore, we can probably disable the quickstart for now.

**Comment by phillip-kruger** [comment]
Cool

**Pull request merged**